core.c

```diff
@@ -648,7 +648,7 @@ VectorNode *malloc_vectorNode() {
   }
   newVectorNode->type = VectorNodeType;
   newVectorNode->refs = refsInit;
-#else	
```

Comment:
Apologies for these whitespace changes that were auto-inserted by my editor. Let me know if I should revert the whitespace for a cleaner change set.